strawberry_django/mutations/resolvers.py

```diff
@@ -552,16 +570,33 @@ def update_m2m(
 
     use_remove = True
     if isinstance(field, ManyToManyField):
-        manager = cast("RelatedManager", getattr(instance, field.attname))
+        remote_field_name = field.attname
+        reverse_field_name = field.remote_field.related_name
```

Comment:
Have some doubts here, probably there is any other way to get reverse relationship name.